mdb/mdb_launch.py

```diff
@@ -90,27 +122,69 @@ def launch(
 
     $ mdb launch -n 8 --auto-restart ./simple-mpi.exe
     """
+
+    numeric_level = getattr(logging, log_level.upper(), None)
+    if not isinstance(numeric_level, int):
+        raise ValueError("Invalid log level: %s" % log_level)
+
+    logging.basicConfig(encoding="utf-8", level=numeric_level)
+    logger = logging.getLogger(__name__)
+
+    MDB_HOME = expanduser(mdb_home)
+    MDB_CERT_PATH = join(MDB_HOME, "cert.pem")
+    MDB_KEY_PATH = join(MDB_HOME, "key.rsa")
+
+    if not exists(MDB_HOME):
+        mkdir(MDB_HOME)
+
+    if not (exists(MDB_CERT_PATH) or exists(MDB_KEY_PATH)):
+        subj = "/C=XX/ST=mdb/L=mdb/O=mdb/OU=mdb/CN=localhost"
```

Comment:
```suggestion
        subj = f"/C=XX/ST=mdb/L=mdb/O=mdb/OU=mdb/CN={hostname}"
```
Should be consistent with the hostname given on the CLI.